chemprop/v2/cli/convert.py

```diff
@@ -0,0 +1,58 @@
+import sys
+import torch
+import logging
+from pathlib import Path
+from argparse import ArgumentParser, Namespace
+
+from chemprop.v2.cli.utils import Subcommand
+from chemprop.v2.utils.v1_to_v2 import convert_model_v1_to_v2_file
+
+logger = logging.getLogger(__name__)
+
+class ConvertSubcommand(Subcommand):
+    COMMAND = "convert"
+    HELP = "convert a v1 model checkpoint (.pt) to a v2 model checkpoint (.ckpt)"
+
+    @classmethod
+    def add_args(cls, parser: ArgumentParser) -> ArgumentParser:
+        return add_convert_args(parser)
+
+    @classmethod
+    def func(cls, args: Namespace):
+        args = process_args(args)
+        main(args)
+
+def add_convert_args(parser: ArgumentParser) -> ArgumentParser:
+    parser.add_argument(
+        "-i",
+        "--input-path",
+        type=str,
+        required=True,
+        help="Path to a v1 model .pt checkpoint file.",
+    )
+    parser.add_argument(
+        "-o",
+        "--output-path",
+        type=str,
+        help="Path at which the converted model will be saved. If not specified, it defaults to the name of the input file suffixed with '_v2.ckpt' in your current working directory.",
+    )
+    return parser
+
+def process_args(args: Namespace) -> Namespace:
+    args.input_path = Path(args.input_path)
+    args.output_path = Path(args.output_path or Path.cwd() / (args.input_path.stem + "_v2.ckpt"))
+    return args
+
+def main(args: Namespace) -> None:
+    logging.info(f"Converting v1 model checkpoint '{args.input_path}' to v2 model checkpoint '{args.output_path}'...")
+    convert_model_v1_to_v2_file(str(args.input_path), str(args.output_path))
+
+if __name__ == "__main__":
+    parser = ArgumentParser()
+    ConvertSubcommand.add_args(parser)
+
+    logger.basicConfig(stream=sys.stdout, level=logging.DEBUG, force=True)
```

Comment:
Sorry Hao-Wei, I should have been more clear. The logging syntax is
`logging.basicConfig(...)`
`logger.info(...)`
These need to be changed on lines 47 and 54.